### ISSUE TITLE
Tooltips cannot be accessed by screen readers in certain locations

### ISSUE THREAD

**Issue by jjromary** [grafana/grafana#57207]
**Steps to reproduce**:
Access pages as shown in the images below
![image](https://user-images.githubusercontent.com/37199044/196510326-c5f53e52-3474-40c9-bb18-fef7a28226e3.png)
![image](https://user-images.githubusercontent.com/37199044/196510396-77321fca-9be6-401f-845b-93bff6eabaed.png)
![image](https://user-images.githubusercontent.com/37199044/196510421-29011ae5-223e-44c2-bef5-d04d778ae40a.png)
![image](https://user-images.githubusercontent.com/37199044/196510438-085474da-a11f-48da-8a01-3bfab104937b.png)

**Actual Result**:
Screen readers cannot access tooltips 

**Expected Result**
Screen readers can access tooltips

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: 9.3.0
- Data source type & version:
- User OS & Browser: Ubutun 20.04 [whats my browser](https://www.whatsmybrowser.org/b/MS67V)
- Others:

**Comment by ashharrison90** [comment]
hi @jjromary 👋 

thanks for raising this! we've been making progress with our accessibility support, i'll add this to our backlog and we'll hopefully get it fixed soon 🙂

**Comment by JoaoSilvaGrafana** [comment]
Thanks for raising this, there is another one of these not accessible tooltips mentioned in this issue: https://github.com/grafana/grafana/issues/47138 so when we fix this let's check that one as well.
